orval.config.ts

```diff
@@ -139,7 +139,7 @@ export default defineConfig({
     },
     input: {
       target:
-        "https://raw.githubusercontent.com/danskernesdigitalebibliotek/dpl-cms/opening-hours-api/openapi.json",
+        "https://raw.githubusercontent.com/danskernesdigitalebibliotek/dpl-cms/opening-hours-repeated/openapi.json",
```

Comment:
This must be set to the `develop` branch after https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1027 is merged